packages/cli-kit/src/public/node/template.ts

```diff
@@ -20,7 +23,7 @@ export function create(templateContent: string) {
  * @param to - Output directory.
  * @param data - Data to feed the template engine.
  */
-export async function recursiveDirectoryCopy(from: string, to: string, data: object) {
+export async function recursiveDirectoryCopy(from: string, to: string, data: object): Promise<void> {
```

Comment:
I'd move this one to `public/node/liquid.ts` and rename it to `recursiveLiquidTemplateCopy`